### PR TITLE
Small grammar update in HttpFoundation

### PR DESCRIPTION
**Pull request by mattjanssen**
"... let you disable buffering at [the] config level or[by] adding ..."

### CODE REVIEW

File: components/http_foundation.rst [hunk 1]
```diff
@@ -473,7 +473,7 @@ represented by a PHP callable instead of a string::
 
     Additionally, PHP isn't the only layer that can buffer output. Your web
     server might also buffer based on its configuration. Some servers, such as
-    Nginx, let you disable buffering at config level or adding a special HTTP
+    Nginx, let you disable buffering at the config level by adding a special HTTP
```
Comment (HeahDude) [inline components/http_foundation.rst]: The `or` has some meaning here and should be kept. One can either disable by configuring the nginx config file, OR add the header at the application level using Symfony API.

Comment (mattjanssen) [reply components/http_foundation.rst]: Gotcha! I'll change it to "or **by**". That's the missing word here then.

**Review by HeahDude — APPROVED** [review]
Thanks!

**Comment by javiereguiluz** [comment]
@mattjanssen thanks for fixing these errors. I love this kind of fixes from native English speakers. Thanks!

**Pull request closed without merging**